en/docs/assets/attachments/InstallAndSetup/apim200_to_apim260_gateway_artifact_migrator_for_tenants.ps1

```diff
@@ -0,0 +1,26 @@
+param (
```

Comment:
Lets rename the file name to "apim200_to_apim300_gateway_artifact_migrator_for_tenants.ps1"

Reply:
Fixed